SayTheirNames/Source/View/Filter/FilterCategoryCell.swift

```diff
@@ -24,60 +24,61 @@
 
 import UIKit
 
-class LocationCell: UICollectionViewCell {
+final class FilterCategoryCell: UICollectionViewCell {
     
-    var titleLabel: UILabel = {
-        let label = UILabel()
-        label.textAlignment = .center
-        return label
-    }()
+    private var titleLabel = UILabel.create {
+        $0.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
This line and lines 61-66 were all accomplished in `titleLabel.fillSuperView()` (L50) before. Is the style you have here preferred? I assumed we should be using those convenience methods.

Reply:
Did this to be consistent with the rest of the code in the diff but will add a quick follow up diff to use `fillSuperView()`